ludwig/data/preprocessing.py

```diff
@@ -1306,6 +1308,10 @@ def _preprocess_file_for_training(
             random_seed=random_seed
         )
 
+        # save split values for use by visualization routines
+        split_fp = os.path.splitext(dataset)[0] + '_split.txt'
```

Comment:
maybe .csv?

Reply:
I can give it the `.csv` suffix.  Do you also want a header row ('split') or a headerless csv file is OK?

Reply:
Actually, I think we should use a headerless csv file because this allows use of the `load_array()` function.